### ISSUE TITLE
Boomstick/Mod/5/C/White Phosphorus Shells converts 50% Kinetic/Explosive to Fire

### ISSUE THREAD

**Issue by phg49389** [phg49389/drg-weapons-calculator#31]
Elythnwaen informed me this morning that WPS adds 50% of Kinetic + Explosive Damage as Heat, and Converts 50% of it to Fire Damage. Need to update the Breakpoints in Boomstick accordingly.

**Comment by phg49389** [comment]
This has been fixed and will be in the new release. Closing issue.

**Comment by phg49389** [comment]
About 8 months ago, while testing this for Gaming Existence, I was able to disprove my original findings. Boomstick T5.C only applies its 50% conversion to Fire + Heat to the Kinetic DamageComponent of the shotgun pellets; NOT to the blastwave! My original tests were wrong because I took it killing a Deeptora Honeycomb in one shot as converting the Blastwave. The problem with that test was because I didn't know about Burning DoT doing its 6 Fire-element damage immediately upon proc'ing, it was pushing over the breakpoint of the oneshot and tricking myself.
